README.md

```diff
@@ -3,6 +3,11 @@
 
 An extensible form builder based on [React JSON Schema Form](https://github.com/rjsf-team/react-jsonschema-form).
 
+## Demos
+
+Basic form playground: https://openfun.github.io/verna/basic
+Poll playground: https://openfun.github.io/verna/poll
```

Comment:
```suggestion
## Demos

To demonstrate verna capabilities, we developed several use case examples.

- Basic form playground:[👀 Live demo](https://openfun.github.io/verna/basic), [🧑‍💻 Source code](https://github.com/openfun/verna/tree/main/examples/playground)
- Poll playground: [👀 Live demo](https://openfun.github.io/verna/basic), [🧑‍💻 Source code](https://github.com/openfun/verna/tree/main/examples/playground)
```

Reply:
I added only the Basic form then for now, i'll had the other one on the pr of grommet